### ISSUE TITLE
Deprecation warning for using 'method_whitelist' with Retry from urllib3

### ISSUE THREAD

**Issue by kdleijer** [snok/django-auth-adfs#187]
Hi,

There is a deprecation warning for using 'method_whitelist' with Retry from urllib3.

```
/django_auth_adfs/config.py:192
DeprecationWarning: Using 'method_whitelist' with Retry is deprecated and will be removed in v2.0. Use 'allowed_methods' instead
```

For more info, have a look at their documentation / code:
- https://pypi.org/project/urllib3/1.26.0/
- https://github.com/urllib3/urllib3/blob/1.26.7/src/urllib3/util/retry.py#L241
- https://github.com/urllib3/urllib3/blob/1.26.7/src/urllib3/util/retry.py#L252

Thanks!

Best regards
Koen de Leijer

**Comment by JonasKs** [comment]
Hello! Thank you so much for a well written issue report😊

Would you be interested in creating a pull request?

**Comment by kdleijer** [comment]
> Hello! Thank you so much for a well written issue reportblush
> 
> Would you be interested in creating a pull request?

Yes I am. Never worked with poetry before, but I will give it a try

**Comment by JonasKs** [comment]
We're happy to help! A short introduction can be found [here](https://github.com/snok/django-auth-adfs/blob/master/CONTRIBUTING.rst#set-up-your-environment). 😊

**Comment by JonasKs** [comment]
Closed by #188
